xdsl/pattern_rewriter.py

```diff
@@ -609,6 +609,51 @@ def match_and_rewrite(self, op: Operation, rewriter: PatternRewriter) -> None:
         return
 
 
+@dataclass(eq=False)
+class Worklist:
+    _lifo: list[Operation | None] = field(default_factory=list, init=False)
+    """
+    The list of operations to iterate over, used as a lifo stack.
```

Comment:
Not sure what you guys' think, but in terms of terminology, I would always call a stack LIFO and a queue FIFO. So, why not just call this the `op_stack` or whatever?